UI/window-basic-main-outputs.cpp

```diff
@@ -832,8 +832,6 @@ bool SimpleOutput::StartStreaming(obs_service_t *service)
 
 static void remove_reserved_file_characters(string &s)
```

Comment:
Is this function necessary? I couldn't find if regular recordings had some similar handling, but this function was only called by the replay start functions. Additionally it was the reason directories were being removed so I just deleted those cases but left the rest.

Reply:
I think it's used for replays because of the fact that replay suffixes (set in advanced) might contain invalid characters.

Reply:
Gotcha, I can see that being in a issue, so I guess it is good that I left most of it intact.